### DISCUSSION TITLE
Cannot delete blob store

### DISCUSSION THREAD

**Discussion by andrewdku** [vercel/community#5227] [Bug Report]
### Page to Investigate

https://vercel.com/dashboard/stores

### Steps to Reproduce

- Create a new Blob database
<img width="524" alt="image" src="https://github.com/vercel/community/assets/140819896/6721bed8-0bc1-4cf2-ac07-08611ceedc70">

- Head to the database's settings
<img width="527" alt="image" src="https://github.com/vercel/community/assets/140819896/a1e6ad2a-1c67-48d7-ab8a-28a2941afcda">

- Scroll down to "Delete Store"
<img width="939" alt="image" src="https://github.com/vercel/community/assets/140819896/1bf70c6c-fb02-42d4-87bb-cbf881d1ed0f">

At this point, I cannot delete the database since it says it must be empty. However, I haven't added anything to it.
<img width="961" alt="image" src="https://github.com/vercel/community/assets/140819896/7c55375e-f7ad-46b0-bb89-d8382e01df2a">

**Comment by luismeyer** [comment]
hey @m1-dev, 
we deployed a fix for this yesterday evening. Would you be so kind to check if it's working for you now?

**Reply by andrewdku** [reply]
@luismeyer It appears to be working as intended. Thanks.
